projects/observability/src/shared/components/cartesian/d3/legend/cartesian-legend.ts

```diff
@@ -166,9 +169,11 @@ export class CartesianLegend<TData> {
 
     this.updateLegendClassesAndStyle();
 
-    legendEntry
-      .on('mouseover', () => legendEntry.select('.filter').style('visibility', 'visible'))
-      .on('mouseout', () => legendEntry.select('.filter').style('visibility', 'hidden'));
+    if (showFilters) {
+      legendEntry
+        .on('mouseover', () => legendEntry.select('.filter').style('visibility', 'visible'))
+        .on('mouseout', () => legendEntry.select('.filter').style('visibility', 'hidden'));
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Element/mouseleave_event

Should this be `mouseleave`?

Reply:
Oh never mind, this is the d3 event.